### PR TITLE
New Codio Mac Cheat Sheet

### PR DESCRIPTION
**Pull request by vikashvverma**
IA Page: https://duck.co/ia/view/codio_mac_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=codio mac cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/codio_mac_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=codio mac cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/codio_mac_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=codio mac cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/codio_mac_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/codio_mac.json [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+			"key": "[⌘] [Z]",
+			"val": "Undo"
+		}, {
+			"key": "[⌘] [\\[]",
+			"val": "Indent less"
+		}, {
+			"key": "[⌘] [\\]]",
+			"val": "Indent more"
+		}, {
+			"key": "[⌘] [⌥] [F]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/codio_mac.json]: I would prefer we use `[Enter]` and `[Shift]` -- I'm concerned these symbols are too small to read.

File: share/goodie/cheat_sheets/json/codio_mac.json [hunk 2]
```diff
@@ -0,0 +1,279 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/codio_mac.json]: @vikashvverma the filename needs to be `codio-mac.json`

File: share/goodie/cheat_sheets/json/codio_mac.json [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+    "aliases": [
+        "codio mac",
+        "codio osx",
+        "codio os x",
+        "codio mac os",
+        "codio apple",
+        "codio mac osx"
+    ],
+	"section_order": [
+		"Editor",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/codio_mac.json]: Please change the tabs to spaces :+1:

File: share/goodie/cheat_sheets/json/codio_mac.json [hunk 4]
```diff
@@ -0,0 +1,279 @@
+{
+	"id": "codio_mac_cheat_sheet",
+	"name": "Codio Shortcuts (Mac)",
+	"description": "The default set of keyboard shortcuts to use Codio efficiently. You can customise these by editing your User preferences, which can be found in Codio > Preferences -> User.",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/codio_mac.json]: "Default keyboard shortcuts for Codio...

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=codio mac cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/codio_mac_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=codio mac cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/codio_mac_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=codio mac cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/codio_mac_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=codio mac cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/codio_mac_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@vikashvverma Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@vikashvverma this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

File: share/goodie/cheat_sheets/json/codio-mac.json [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+	"template_type": "keyboard",
+    "aliases": [
+        "codio osx",
+        "codio os x",
+        "codio mac os",
+        "codio apple",
+        "codio mac osx"
+    ],
+	"section_order": [
+		"Editor",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/codio-mac.json]: @vikashvverma looks like you have tabs in here 😦  -- Please switch to 4-space soft tabs

**Pull request merged**